examples/mlperf/model_spec.py

```diff
@@ -22,9 +22,10 @@ def spec_retinanet():
 
 def spec_unet3d():
   # 3D UNET
-  from models.unet3d import UNet3D
+  from models.unet3d_v2 import UNet3D
```

Comment:
Is this right?

Reply:
this should be just unet3d, I forgot to update it when I renamed the files